Mathlib/SetTheory/Ordinal/FixedPointApproximants.lean

```diff
@@ -192,7 +192,8 @@ theorem lfpApprox_le_of_mem_fixedPoints {a : α}
       rw [h_y]
       exact h_le_init
 
-/-- The least fixed point of f is reached after the successor of the domains cardinality -/
+/-- The least fixed point of f is reached at the ordinal-successor index
+  of the domains cardinality -/
```

Comment:
```suggestion
/-- The least fixed point of f is reached at the successor
  of the domain's cardinality -/
```
I'd argue this theorem says something slightly distinct, which is that the approximants of `⊥` specifically converge at the least fixed point.

Reply:
I rephrased that sentence to include the convergence at the successor starting from `⊥`.